client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -565,6 +518,66 @@ private static WebServiceFeature newMemberSubmissionAddressingFeature() {
         }
     }
 
+    private static WebServiceFeature soapClientSetForceDoclitBare() {
+        ////  when client.command is executed if doclit.bare is not set then this exception occurs:
+        ////  Unexpected element {http://schemas.microsoft.com/wbem/wsman/1/windows/shell}CommandResponse found.
+        ////  Expected {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}CommandResponse
+        //  serviceInfo.setProperty("soap.force.doclit.bare", true);
+        //  serviceInfo.setProperty("soap.no.validate.parts", true);
+        return new AbstractFeature() {
+            @Override
+            public void initialize(Client client, Bus bus) {
+                client.getEndpoint().getEndpointInfo().getService().setProperty("soap.force.doclit.bare", true);
+                super.initialize(client, bus);
+            }
+        };
+    }
+
+    private WebServiceFeature soapClientAsyncHttpConduitParameters() {
+        return new AbstractFeature() {
+            @Override
+            public void initialize(Client client, Bus bus) {
+                AsyncHTTPConduit httpClient = (AsyncHTTPConduit) client.getConduit();
+
+                if (disableCertificateChecks) {
+                    TLSClientParameters tlsClientParameters = new TLSClientParameters();
+                    tlsClientParameters.setDisableCNCheck(true);
+                    tlsClientParameters.setTrustManagers(new TrustManager[]{new X509TrustManager() {
```

Comment:
(just an FYI that that exists in brooklyn -- np as you've written it, and certainly not suggesting adding a dep to brooklyn utils just for that)